From e2f4f042d0872089257b346015e9b03e8247338f Mon Sep 17 00:00:00 2001
From: Viktor Szakats <vszakats@users.noreply.github.com>
Date: Tue, 21 Feb 2017 23:01:37 -0500
Subject: [PATCH 3/3] urldata: include curl_sspi.h when Windows SSPI is enabled

f77dabe broke builds in Windows using Windows SSPI but not Windows SSL.

Bug: https://github.com/curl/curl/issues/1276
Reported-by: jveazey@users.noreply.github.com
---
 lib/urldata.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/urldata.h b/lib/urldata.h
index 648b3e8..7f87913 100644
--- a/lib/urldata.h
+++ b/lib/urldata.h
@@ -136,8 +136,10 @@
 #undef realloc
 #endif /* USE_AXTLS */
 
-#ifdef USE_SCHANNEL
+#if defined(USE_SCHANNEL) || defined(USE_WINDOWS_SSPI)
 #include "curl_sspi.h"
+#endif
+#ifdef USE_SCHANNEL
 #include <schnlsp.h>
 #include <schannel.h>
 #endif
-- 
2.11.1.windows.1.dirty

